### DISCUSSION TITLE
Blur background

### DISCUSSION THREAD

**Discussion by truongvan1003** [kovidgoyal/kitty#6390] [Ideas]
I see that, this one is a feature very important to improve user experience. I can support or pay to you can develop this feature.

**Comment by hungvx-dev** [comment]
I saw him merge this feature https://github.com/kovidgoyal/kitty/commit/7a1bdb4ff1eec2d369ce40f56c3dc9c5e4583bd8. That's so great. @kovidgoyal  Thank you so much. When it can release on homebrew? Can you release it now?

**Reply by kovidgoyal** [reply]
Use the kitty nightly builds.

**Reply by hungvx-dev** [reply]
@kovidgoyal  I don't see this feature release on nightly version https://github.com/kovidgoyal/kitty/releases/tag/nightly. 
background_blur not work
